internal/datanode/data_node.go

```diff
@@ -1194,53 +1197,91 @@ func importFlushReqFunc(node *DataNode, req *datapb.ImportTaskRequest, res *root
 				zap.Int("# of channels", len(chNames)),
 				zap.Strings("channel names", chNames),
 			)
-			return fmt.Errorf("syncSegmentID Failed: invalid shard ID %d", shardID)
+			return 0, "", fmt.Errorf("syncSegmentID Failed: invalid shard ID %d", shardID)
 		}
 
 		tr := timerecord.NewTimeRecorder("import callback function")
```

Comment:
this is no longer time record for import callback func?

Reply:
illegal shardID, return error, this task will failed. no need to record time